src/components/ProfileButtons/index.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
+
+import Icon from "components/shared/Icon";
+import getIcon from "util/getIcon";
+import { MovieButton, ButtonHolder } from "./styles";
+import API from "api/moviedb.instance";
+import { useAuth } from "contexts/AuthContext";
+import Alert from "components/shared/Alert";
+
+export default function ProfileButtons({ openTrailer, id, type }) {
+    const media = type === "movie" ? "Movies" : "TV";
+    const {
+        state: { user, session, userMedia },
+    } = useAuth();
+    const [isFavorite, setIsFavorite] = useState(false);
+    const [isListed, setIsListed] = useState(false);
+    const [isDisplayed, setIsDisplayed] = useState({
+        display: false,
+        message: "",
+    });
+
+    const config = {
+        headers: {
+            "Content-Type": "application/json;charset=utf-8",
+        },
+        params: {
+            session_id: session?.session_id,
+        },
+    };
+
+    const favoriteBody = {
+        media_type: type,
+        media_id: id,
+        favorite: !isFavorite,
+    };
+    const listBody = {
+        media_type: type,
+        media_id: id,
+        watchlist: !isListed,
+    };
+
+    useEffect(() => {
+        const favorites =
```

Comment:
Can you format it like this? Wherein the operatores such as `&&` is in the front. Just a suggestion cause linters like ESLint takes this as a formatting error.

```javascript
const favorites = userMedia
  && userMedia[`favorite${media}`].results.map((item) => item.id);
```